packages/search-ui/src/hooks/usePosNegTracking.ts

```diff
@@ -0,0 +1,43 @@
+import { ClickTracking, PosNegTracking, ResultClickedFn, Token } from '@sajari/react-hooks';
+import { isFunction, noop } from '@sajari/react-sdk-utils';
+import { PosNegLocalStorageManager } from '@sajari/sdk-js';
+
+import { ResultValues } from '../Results/types';
+
+export interface UseNTrackingParams {
```

Comment:
Note here if it was not intentional
```suggestion
export interface UsePosNegTrackingParams {
```